packages/plugin-assets/src/init.js

```diff
@@ -0,0 +1,78 @@
+const { TwilioServerlessApiClient } = require('@twilio-labs/serverless-api');
+const {
+  createService,
+} = require('@twilio-labs/serverless-api/dist/api/services');
+const {
+  createEnvironmentFromSuffix,
+  getEnvironment,
+} = require('@twilio-labs/serverless-api/dist/api/environments');
+const { TwilioCliError } = require('@twilio/cli-core').services.error;
+
+const DEFAULT_ASSET_SERVICE_NAME = 'CLI-Assets-Bucket';
+
+const createServiceAndEnvironment = async client => {
```

Comment:
Any reason why you use arrow functions instead of regular async function declarations?

Reply:
No reason, have updated to async function declarations.